webapp/app/assessment/fra/sections/extentOfForest/extentOfForestValidatorState.js

```diff
@@ -28,7 +28,7 @@ export const areasNotExceedingTotalLandAreaValidator = datum => state => {
   return greaterThanOrEqualTo(otherLand, 0)
 }
 
-//==== a
+//= === a
```

Comment:
?

Reply:
Maybe prettier or eslint does this, but it is a bit unconsistent

Reply:
removed